src/main/groovy/org/kordamp/gradle/plugin/jandex/JandexExtension.groovy

```diff
@@ -28,4 +29,13 @@ interface JandexExtension {
     Property<String> getVersion()
 
     Property<Integer> getIndexVersion()
+
+    /**
+     * Additional task names or patterns that should depend on the jandex task.
+     * The plugin already sets up dependencies for 'compileTestJava', 'jar', and Javadoc tasks.
+     * This property allows adding more tasks that should depend on the jandex task.
+     * You can specify exact task names or patterns using Ant-style patterns (e.g., '*Test').
+     * @return a list of task names or patterns
+     */
```

Comment:
Now this is a strange change. This inverts the relationship between jandex and tasks that depends on it. 

Instead of explicitly setting that a task depends on jandex, this tells jandex to set itself as a dependency on a target task.